src/coreclr/jit/importercalls.cpp

```diff
@@ -2540,25 +2540,6 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
         return new (this, GT_LABEL) GenTree(GT_LABEL, TYP_I_IMPL);
     }
 
-    switch (ni)
-    {
-        // CreateSpan must be expanded for NativeAOT
-        case NI_System_Runtime_CompilerServices_RuntimeHelpers_CreateSpan:
-        case NI_System_Runtime_CompilerServices_RuntimeHelpers_InitializeArray:
-            mustExpand |= IsTargetAbi(CORINFO_NATIVEAOT_ABI);
-            break;
-
-        case NI_Internal_Runtime_MethodTable_Of:
-        case NI_System_Activator_AllocatorOf:
-        case NI_System_Activator_DefaultConstructorOf:
-        case NI_System_EETypePtr_EETypePtrOf:
-            mustExpand = true;
-            break;
-
-        default:
-            break;
-    }
-
     // Allow some lighweight intrinsics in Tier0 which can improve throughput
```

Comment:
`betterToExpand` is applicable to more than just Tier0 now. This comment should be updated.

Reply:
I'm not sure I understand - the comment is still applicable to the immediately following block. I specifically moved the native aot stuff after the block so that I don't break the flow there. Native aot has its own comment block.

Reply:
Yes, the comment is still applicable to the immediately following block. It does not explain the full purpose of `bool betterToExpand` anymore. I was just asking to move the comment to the immediately following block (and maybe adjust it a bit for the new context).